### ISSUE TITLE
Identities and Credentials

### ISSUE THREAD

**Issue by elf-pavlik** [gobengo/dom-activity#1]
Hi,

Have you taken a look at work happening in [Credentials CG](https://www.w3.org/community/credentials/)

e.g. early draft of [Identity Credentials](http://opencreds.org/specs/source/identity-credentials/)

**Comment by gobengo** [comment]
@elf-pavlik No, I hadn't seen that, but thanks for the share. I've been PMing an OpenIDConnect Provider as of late ([staging RP link](https://lfoidc-rp-staging.herokuapp.com)), and I'm pretty happy with the protocol, so it's interesting to imagine how these could work together, and I was happy to see at least [a mention](http://opencreds.org/specs/source/identity-credentials/#integration-with-openid-oauth1-and-oauth2) of it in Issue #4.

wrt DOMActivity, it looks like an integration point would be in making sure the Identity Credentials could be easily leveraged to create 'Actor' objects for DOMActivity, when the actor is meant to be the end-user of the user-agent.

I'll see you in freenode/#credentials and the credentials mailing list!

Any other takeaways or relations I should be sure to think about?
